src/pages/theme/vars.css

```diff
@@ -0,0 +1,33 @@
+:root {
+  /* Base16 Color Palette */
+  --base00: #000000; /* Default Background */
+  --base01: #18181b; /* Lighter Background */
+  --base02: #27272a; /* Selection Background */
+  --base03: #202125; /* Line Highlighting */
+  --base04: #6b7280; /* Dark Foreground */
+  --base05: #9ca3af; /* Default Foreground */
+  --base06: #e5e7eb; /* Light Foreground */
+  --base07: #ffffff; /* Light Background */
+  --base08: #ff0000; /* Accent Color */
+  --base09: #ffa500; /* Accent Color */
+  --base0A: #ffff00; /* Accent Color */
+  --base0B: #00ff00; /* Accent Color */
+  --base0C: #00ffff; /* Accent Color */
+  --base0D: #06b6d4; /* Primary Color */
+  --base0E: #ff00ff; /* Accent Color */
+  --base0F: #a52a2a; /* Accent Color */
+
+  /* Component-Specific Variables */
+  --peersky-background-color: var(--base00);
+  --peersky-p2p-background-color: var(--base01);
+  --peersky-text-color: var(--base07);
+  --background-nav: var(--base02);
```

Comment:
IMO we should use a prefix like `--browser-theme-background-nav` so that there's less chance of conflicting with the styles on a page.

Reply:
Oh yes!